go/felix/rules/static.go

```diff
@@ -429,6 +450,87 @@ func (r *DefaultRuleRenderer) StaticNATOutputChains(ipVersion uint8) []*Chain {
 	}}
 }
 
+func (r *DefaultRuleRenderer) StaticRawTableChains(ipVersion uint8) []*Chain {
+	return []*Chain{
+		r.failsafeInChain(),
+		r.failsafeOutChain(),
```

Comment:
Hmm, not sure; and sounds like you're not sure too.  Should we leave some kind of reminder to come back to this?